### PR TITLE
feat: Added not found page

### PR DESCRIPTION
**Pull request by bharati-21**
This PR is for the 404 not found page that allows users to navigate back to the home page.

### CODE REVIEW

File: src/routes/WebsiteRoutes.jsx [hunk 1]
```diff
@@ -17,6 +18,7 @@ const WebsiteRoutes = () => {
             <Route path="/login" element={<Login />} />
             <Route path="/mockman" element={<Mockman />} /> 
             <Route path="/" element={<LandingPage />} /> 
+            <Route path="*" element={<NotFound />} />
             <Route path="/" element={<ProtectedRoutes />}>
                 <Route path="/cart" element={<Cart />} />
                 <Route path="/wishlist" element={<WishList />} />
```
Comment (rahulyadav139) [inline src/routes/WebsiteRoutes.jsx:24]: Routing looks good to me.  You are planning to protect routes later if I am not wrong? Other than that everything is clean and readable.

**Review by rahulyadav139 — APPROVED** [review]

File: src/routes/WebsiteRoutes.jsx (hunk 1 above)
Comment (bharati-21) [reply src/routes/WebsiteRoutes.jsx:24]: Thank you. No, I have already added private routes.

Comment (bharati-21) [reply src/routes/WebsiteRoutes.jsx:24]: Thank you for your review!

**Review by viru2001 — APPROVED** [review]

**Review by viru2001 — COMMENTED** [review]
looks good

**Pull request merged**